src/PrestaShopBundle/Resources/config/routing/admin/sell/customer_service/merchandise_return.yml

```diff
@@ -43,8 +47,57 @@ admin_order_returns_update:
   defaults:
     _controller: PrestaShopBundle:Admin/Sell/CustomerService/MerchandiseReturn:edit
     _legacy_controller: AdminReturn
-    #    _legacy_link: AdminReturn:updateorder_return
+    _legacy_link: AdminReturn:updateorder_return
+    _legacy_feature_flag: order_return
     _legacy_parameters:
       id_order_return: orderReturnId
   requirements:
     orderReturnId: \d+
+
+admin_order_returns_products_filter:
+  path: /{orderReturnId}/edit
```

Comment:
Be careful you may have conflicts with `admin_order_returns_edit` which has the same path and also handles the POST method, if the POST request always has to pass through this route then you should remove the method from the `admin_order_returns_edit` route

Also it's a detail but if you can place this route right under `admin_order_returns_edit` it will be easier to read/understand

Reply:
Good point! Changed edit to only use GET